out/.gitignore

```diff
@@ -1 +0,0 @@
-*
```

Comment:
Hm, I do think we *want* to ignore the class files. :smile:

Right now various class files from `out/` got added via this PR, and they shouldn't be (because they are generated files).

Please re-ignore `out/` and re-`git rm` the class files from source control :smile: